internal/cmd/gocdk/static/_assets/provision/blob/gcsblob/gcsblob.tf

```diff
@@ -0,0 +1,30 @@
+# TODO(rvangent): Add comments explaining.
```

Comment:
Is there a specific issue tracking this todo already? If not, can you make an issue (or a few) for filling in this group of explanatory comments (or other chunks of commentary that make sense to tackle together?

Reply:
Done.